Argo.xcworkspace/xcshareddata/Argo.xcscmblueprint

```diff
@@ -4,13 +4,14 @@
 
   },
   "DVTSourceControlWorkspaceBlueprintWorkingCopyStatesKey" : {
-    "57C931977B7D2307CC013C2BD93F90CF7C676790" : 0,
+    "194A7A54F3CE0138A6C320E0643B7C60F8218891" : 0,
```

Comment:
This file shouldn't be checked in at all. I'm planning on removing it at some point.